calendar/page.js

```diff
@@ -194,7 +197,8 @@ function getGristOptions() {
 }
 
 
-function setMonthNameInDom(){
+function updateUIAfterNavigation(){
+  //update name of the month and year displaed on the top of the widget
```

Comment:
displaed -> displayed
[also spaces after // would save me time deciding whether it is a big enough deal to mention :)]